images/hub/jupyterhub_config.py

```diff
@@ -51,9 +51,13 @@ def get_config(key, default=None):
 
 c.KubeSpawner.singleuser_extra_labels = get_config('singleuser.extra-labels', {})
 
-# FIXME: Make this better? 
-c.KubeSpawner.singleuser_uid = 1000
-c.KubeSpawner.singleuser_fs_gid = 1000
+c.KubeSpawner.singleuser_uid = get_config('singleuser.uid')
+c.KubeSpawner.singleuser_fs_gid = get_config('singleuser.fs-gid')
+
+# FIXME: Make this the default? This lets us do + with lists for
```

Comment:
This is fine, though you can use `.extend` on config values without initializing them and it figures out that it's a list.